src/System.Device.Gpio/Interop/Unix/Libc/Interop.mmap.cs

```diff
@@ -88,4 +88,8 @@ internal unsafe struct RegisterView
     ///<summary>GPIO Pin Pull-up/down Enable Clock, 2x32 bits, R/W.</summary>
     [FieldOffset(0x98)]
     public fixed uint GPPUDCLK[2];
+
+    /// <summary>Pull-up/down Enable on the BCM2711, 4x32 bits, R/W (2 bits per Pin)</summary>
+    [FieldOffset(0xE4)]
+    public fixed uint GPPUPPDN[4];
```

Comment:
Is this available on other Pis or only on Pi4? Also, do you need access to '/dev/mem' memory mapped file or is this register available via '/dev/gpiomem'?

Reply:
I do not know what happens when you use that register on an older Pi, probably nothing you want. 
I did not change the register map start detection logic, so it uses /dev/gpiomem by default (which works). When forcing it to use /dev/mem, it seems to work as well (at least when sudo'ing).

Reply:
It is definitely something we will want to check as we don't want to map stuff to memory which doesn't exist. In general I would probably propose having another registerView for Pi4

Reply:
Yes, agree. I finally found the Pi3 register map specification: https://www.raspberrypi.org/app/uploads/2012/02/BCM2835-ARM-Peripherals.pdf (pages 89ff). This ends at address B0. 
The corresponding specification document for the BCM2711 still was not published and I took the addresses for the additional registers from https://github.com/joan2937/pigpio/blob/master/pigpio.c (at the top of the file).